tf2onnx/graph.py

```diff
@@ -1134,16 +1139,22 @@ def dump_node_statistics(self):
         return op_cnt
 
     @staticmethod
-    def remove_input(node, to_be_removed):
+    def remove_input(node, to_be_removed, i=None):
```

Comment:
Might be cleaner to do remove_input_at_index and replace_input_at_index instead of making i an optional argument.

Reply:
Renamed i into input_index. Not sure we should create another function which does almost the same thing.

Reply:
That works too.